src/image-record.ts

```diff
@@ -12,18 +12,152 @@
 */
 
 import FilterTransform from './filters/filter-transform';
+import {IListRecord, imageDB} from './image-db';
+
+enum ImageState {
+  NotLoaded,
```

Comment:
What do you think about UPPERCASING these enum states?

Reply:
OK, sure. I had a look around, and it seems that while the official TS examples all use UpperCamelCase, the Google style guide uses UPPERCASE.